tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -541,6 +553,29 @@ class ScopedFilterDescriptor {
   SE_DISALLOW_COPY_AND_ASSIGN(ScopedFilterDescriptor);
 };
 
+// A helper class to decide whether to enable the TENSOR_OP_MATH math type
+template <bool DefaultFlag>
+class TensorOpMath {
+ public:
+  static bool IsEnabled() {
+    static bool is_enabled = IsEnabledImpl();
+    return is_enabled;
+  }
+
+ private:
+  static bool IsEnabledImpl() {
+    const char* tf_env_var_val = getenv("TF_ENABLE_TENSOR_OP_MATH");
```

Comment:
Add a TODO to refactor read from env-var into its own helper function, similar to what TF does. Or reuse the one from TF.

Reply:
Refactored.